Pod/Classes/NotificationConvenience.swift

```diff
@@ -17,6 +17,13 @@ public extension NotificationCenter {
         }
     }
     
+    // Needed for objc compatibility
+    public func post(notification: Notification) -> Void {
+        DispatchQueue.main.async { [weak self] in
```

Comment:
Would you want to include a Bool parameter similar to the old method to specify main thread?